example_template.md

```diff
@@ -0,0 +1,117 @@
+**Table of Contents**
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+- [Run the EXAMPLE NAME](#run-the-example-name)
+  - [🗝️ Pre requirements](#Pre-requirements)
+  - [🔮 Overview of the files](#Overview-of-the-files)
+  - [🏃 Run the Flows](#run-the-flows)
+  - [🌀 Flow Diagram](#flow-diagram)
+  - [🌟 Results](#results)
+  - [🧞 OPTIONAL: EXTRA KNOWLEDGE FROM THIS EXAMPLE](#optional)
+  - [💫 Deploy with Docker](#Deploy-with-docker)
+  - [Next steps](#next-steps)
+  - [Community](#community)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Run the EXAMPLE NAME
+
+Write a short description of
+1. What is this?
+2. What is the expected outcome?
+3. What the user will learn?
+
+If you want to run the example with Docker, check the instruction at the end of this README.  
+
+## 🗝️ Pre requirements
+
+Outline in bullet points anything the user is expected to have before diving in. For example:
+
+1. You have a working Python 3.8 environment. 
+2. We recommend creating a [new python virtual envoriment](https://docs.python.org/3/tutorial/venv.html) to have a clean install of Jina and prevent dependency clashing.   
+3. You have at least 8GB of free space on your hard drive. 
+
+### Install requirements
+
+Begin by cloning the repo so you can get the required files and datasets:
+
+```sh
+git clone https://github.com/jina-ai/examples
+````
+
+And enter the correct folder:
+
+```sh
+cd examples/example_to_use
```

Comment:
we need to tell user that change `example_to_use` to the corresponding name of the folder? To avoid someone directly copy-paste and run this command

Reply:
This is a template for the devs who will write the documentation not really the users

Reply:
Oh ok, great!